cmd/opm/index/add.go

```diff
@@ -55,10 +55,12 @@ func addIndexAddCmd(parent *cobra.Command) {
 	}
 	indexCmd.Flags().Bool("skip-tls", false, "skip TLS certificate verification for container image registries while pulling bundles")
 	indexCmd.Flags().StringP("binary-image", "i", "", "container image for on-image `opm` command")
-	indexCmd.Flags().StringP("container-tool", "c", "podman", "tool to interact with container images (save, build, etc.). One of: [docker, podman]")
+	indexCmd.Flags().StringP("build-tool", "c", "podman", "tool to build container images (build). One of: [docker, podman]")
+	indexCmd.Flags().StringP("pull-tool", "p", "none", "tool to pull container images (save, pull, etc.). One of: [none, docker, podman]")
 	indexCmd.Flags().StringP("tag", "t", "", "custom tag for container image being built")
 	indexCmd.Flags().Bool("permissive", false, "allow registry load errors")
 	indexCmd.Flags().StringP("mode", "", "replaces", "graph update mode that defines how channel graphs are updated. One of: [replaces, semver, semver-skippatch]")
+	indexCmd.Flags().String("container-tool", "", "container-tool has been deprecated. Please select a build and pull tool separately.")
```

Comment:
Note for later: cobra has a way to mark options as deprecated:
```go
indexCmd.Flags().StringP("container-tool", "c", "podman", "tool to interact with container images (save, build, etc.). One of: [docker, podman]")
if err := rootCmd.Flags().MarkDeprecated("container-tool", "ignored in favor of separate build-tool and pull-tool options"); err != nil {
		logrus.Panic(err.Error())
}
```

This has some nice properties like preserving the help when referenced, but omitting it from the standard help output.

Reply:
Ah, nice. I'll update this to include it.